server/controllers/requestController.js

```diff
@@ -0,0 +1,44 @@
+import RequestHelper from '../helpers/requestHelper';
+import users from '../helpers/authHelpers';
+
+/**
+ * This class contains all methods
+ * required to approve/reject
+ *  atrip requests.
+ */
+class RequestController {
+  /**
+   * This method approve the trip request.
+   * @param {object} req The user's trip request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the trip.
+   */
+  static async approveRequest(req, res) {
+    const reqq = await RequestHelper.findRequest('id', parseInt(req.params.id, 10));
+
+    if (!reqq) {
```

Comment:
if this condition passes, what will happen to line 26? It will still run, a nightmare. fix this